mlrun/__init__.py

```diff
@@ -35,7 +35,7 @@
 from .execution import MLClientCtx
 from .model import RunObject, RunTemplate, new_task
 from .package import ArtifactType, DefaultPackager, Packager, handler
-from .platforms import VolumeMount, auto_mount, mount_v3io, v3io_cred
+from .platforms import VolumeMount, auto_mount
```

Comment:
This will break backwards-compatibility for user's existing code. Anyone currently doing something like -
```
import mlrun
function.apply(mlrun.mount_v3io())
```
Will have broken code, and will need to adjust to the new import path.
So, unless there's a specific reason why this import cannot be here, these imports are needed here (instead of importing from `.platforms` adjust to the new location).

Reply:
I didn't realize these were imported by our users from there. I added them back. Thanks, Saar!